linux_os/guide/system/network/network-wireless/wireless_software/wireless_disable_interfaces/bash/shared.sh

```diff
@@ -1,3 +1,7 @@
 # platform = multi_platform_all
 
-nmcli radio all off
+if 
+    rpm -qa | grep NetworkManager
+then
+    nmcli radio all off
+fi
```

Comment:
Could add an `else` clause and print a message stating the reason why the rule was not remediated?

For inspiration:
https://github.com/ComplianceAsCode/content/blob/2f1198047c72452876d7cc12d59204cd479c8b06/linux_os/guide/system/accounts/enable_authselect/bash/shared.sh#L11
https://github.com/ComplianceAsCode/content/blob/2f1198047c72452876d7cc12d59204cd479c8b06/linux_os/guide/system/accounts/enable_authselect/ansible/shared.yml#L29

https://github.com/ComplianceAsCode/content/blob/2f1198047c72452876d7cc12d59204cd479c8b06/linux_os/guide/system/software/integrity/fips/enable_fips_mode/bash/shared.sh#L11